lib/feedback_form.dart

```diff
@@ -56,10 +58,13 @@ class _FeedbackFormState extends State<FeedbackForm> {
               });
             },
           ),
-          RaisedButton(
+          emailVerified ? RaisedButton(
```

Comment:
The whole form should disappear not just the 'Submit' button.

Reply:
okay mista